riff-raff/app/views/snippets/recordTable.scala.html

```diff
@@ -34,9 +34,7 @@
         <tr class="rowlink@if(record.hasWarnings){ has-warnings}">
             <td><time class="makeRelativeDate" withinhours="24" datetime="@record.time">@utils.DateFormats.Medium.print(record.time)</time></td>
             <td><span class="label label-default">@record.deployerName</span></td>
-            @if(allColumns) {
-                <td><a href="@routes.DeployController.viewUUID(record.uuid.toString)" class="rowlink">@record.buildName</a></td>
-            }
+            <td><a href="@routes.DeployController.viewUUID(record.uuid.toString)" target="@if(popOutLink){ _blank }" class="rowlink">@record.buildName</a></td>
```

Comment:
I'm not seeing this on CODE, but the change looks ok so presumably another branch is up there now?

Reply:
I looked into this more and I'd actually made a big boo-boo... it was actually putting in the full project name in the middle (don't know how I missed it), I've now wrapped the contents `Status` cell in the `<a>` instead so that there are no extra `<td>` which means the cell containing the `@record.buildName` can remain conditional (`@if(allColumns)`)